stores/pg/src/storage/index.ts

```
@@ -138,14 +142,12 @@ export class PostgresStore extends MastraStorage {
           ssl: config.ssl,
           max: config.max,
           idleTimeoutMillis: config.idleTimeoutMillis,
-        } as any;
+        };
       } else {
         // This should never happen due to validation above, but included for completeness
-        this.#config = {
-          ...(config as ClientConfig),
-          max: (config as any).max,
-          idleTimeoutMillis: (config as any).idleTimeoutMillis,
-        } as any;
+        throw new Error(
```

we should throw a mastra error here instead